test/fixtures/offering/test-10.js

```diff
@@ -0,0 +1,25 @@
+const { testData: goodData } = require("./test-00-valid-data");
+
+module.exports = {
+  inputType: "UpdateDealInput",
+  isValid: false,
+  // message:
+  //   'Expected a value of type `AWSJSON` for `sections.0.info.0.content.json` but received `"Unquoted String"`.',
+  testData: {
+    ...goodData,
```

Comment:
I about had a heart attack thinking this var name was in the code and not a test lol

Reply:
You can't use qewl-forms?  You make sure to use the goodData?